File: php/src/common/Minnestrone.php
<?

require_once 'Soup.php';


class Minnestrone extends Soup
{

	function __construct()
	{
      $this->soupName = "Minnestrone";
      $this->soupIngredients = [
        "1 Pound tomatos",
        "1/2 cup pasta",
        "1 cup tomato juice",
      ];
    }

}

File: python/src/common/Minnestrone.py

define ['Creational/Abstract_Factory/Soup',
],(
Soup
) ->

	class Minnestrone extends Soup
		constructor : ->
			@soupName = 'Minnestrone'
			@soupIngredients = [
				'1 Pound tomatos',
				'1/2 cup pasta',
				'1 cup tomato juice',
			]

	return Minnestrone
